rdm/0011-creators-contributors-metadata.md

```diff
@@ -0,0 +1,292 @@
+- As an InvenioRDM hoster, I want to use a schema that is compatible with
+  DataCite schema (and others), so that I can inter-operate with other metadata
+  formats.
+- As an InvenioRDM developer, I want to code to a common metadata
+  schema, so that I don't repeat code and so that I benefit from and to others'
+  work.
+- As a researcher, I want to cite a record with its authors/creators,
+  so that my citation follows standards.
+- As a researcher, I want a DOI for my record which is only possible
+  if the equivalent `Creator` field is passed to the minting authority (DataCite).
+- As an InvenioRDM hoster, I want to provide a generic name field, so that
+  international authors and contributors can enter their full name in their own
+  way.
+- As an InvenioRDM hoster, I want to support person and organization identifiers,
+  so that I can disambiguate between entities.
+
+
+## Detailed design
+
+Creators and contributors can be people or organizations. Their schema structure
+will be reused. Their definition in the `"definitions"` section is:
+
+```json
+{
+  "definitions": {
+    "identifier": {
+      "type": "object",
+      "description": "A generic identifier.",
+      "additionalProperties": false,
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "Kind of identifier.",
+          "enum": [
+            "ads",
+            "ark",
+            "arxiv",
+            "bioproject",
+            "biosample",
+            "doi",
+            "ean13",
+            "ean8",
+            "ensembl",
+            "genome",
+            "gnd",
+            "hal",
+            "handle",
+            "isbn",
+            "isni",
+            "issn",
+            "istc",
+            "lsid",
+            "orcid",
+            "pmcid",
+            "pmid",
+            "purl",
+            "refseq",
+            "ror",
+            "sra",
+            "uniprot",
+            "url",
+            "urn"
+          ]
+        },
+        "value": {
+          "type": "string"
+        }
+      },
+      "required": ["type", "value"]
+    }
+  },
+  "person_or_org": {
+    "type": "object",
+    "description": "A person or organization.",
+    "additionalProperties": false,
+    "properties": {
+      "affiliation": {
+        "type": "string",
+        "description": "Affiliation for the purpose of the specific record."
+      },
+      "email": {
+        "type": "string",
+        "description": "Contact email for the purpose of this specific record.",
+        "format": "email"
+      },
+      "full_name": {
+        "type": "string",
+        "description": "Full name of person or organization. Personal name format: family, given."
+      },
+      "ids": {
+        "type": "array",
+        "description": "List of IDs related with the person or org. (e.g., ORCID, RORid)",
+        "uniqueItems": true,
+        "items": { "$ref": "#/definitions/identifier" }
```

Comment:
I don't know if this syntax is possible to be honest. Maybe `"one_of"` is necessary...

Reply:
It looks right, but I haven't tried it before. I'll add up a validation test of the schema to my to-do list.